README.md

```diff
@@ -17,7 +17,7 @@ Example:
 </paper-toolbar>
 ```
 
-`paper-toolbar` has a standard height, but can made be taller by setting `tall`
+`paper-toolbar` has a standard height, but can made be taller by setting `.tall`

```

Comment:
We shouldn't change it to `.tall` class.  I don't think any of our docs are doing that.